### PR TITLE
release v0.1.4 - support sticky sessions and add RBAC

### PR DESCRIPTION
**Pull request by andrewsykim**
cc @xmudrii @bhcleek 

Fixes https://github.com/digitalocean/digitalocean-cloud-controller-manager/issues/44

### CODE REVIEW

File: examples/loadbalancers/http-with-sticky-sessions.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
```
Comment (andrewsykim) [inline examples/loadbalancers/http-with-sticky-sessions.yml]: screen shot 2018-03-18 at 6 52 02 pm

Comment (xmudrii) [reply examples/loadbalancers/http-with-sticky-sessions.yml]: Is this the behavior?

Comment (andrewsykim) [reply examples/loadbalancers/http-with-sticky-sessions.yml]: Yeah! Just putting it here to show what the sticky session config will look like based on the example

Comment (xmudrii) [reply examples/loadbalancers/http-with-sticky-sessions.yml]: I'm happy to see it's working as expected! :blush:

Comment (andrewsykim) [reply examples/loadbalancers/http-with-sticky-sessions.yml]: Same! Thanks again for the contribution :)

**Pull request merged**